whist/server/api/game/trick.py

```diff
@@ -55,9 +55,11 @@ def play_card(game_id: str, card: Card, background_tasks: BackgroundTasks,
     :return: the stack after card being played if successful. If not the players turn raises error.
     """
     room = game_service.get(game_id)
-
+    trick = room.current_trick()
+    if trick.done:
+        trick = room.next_trick()
+        background_tasks.add_task(channel_service.notify, game_id, TrickStartedEvent())
```

Comment:
Does this make sense from the client's point of view?
Does one even know their cards before the next trick has started?

Reply:
Yes, a trick is playing a card by each player. For the exception of the last trick, this is valid, but there is still a bug in the Core https://github.com/Whist-Team/Whist-Core/pull/213

I think we discussed this a while ago. We want to start the next trick automatically, once the winner of the current trick is playing its next card.

Reply:
Would it not be better to start the next trick after the last card of the current trick has been played?
So clients can always know and show the correct information

Reply:
By Whist Rule you are only allowed to see the current Trick. So if the last card was played and the next trick has been started immediately could result in problems that other players wouldn't have a chance to see the last card.

Reply:
This can be handled on the client side.
Send a PlayCardEvent (with the card) and a TrickDoneEvent (with the winner).
The client will show the card being played and then mark the winner somehow. Then the next player to play is notified and they can play a card which will remove the visible last trick from the "board".

Reply:
We want as much logic as possible on the server side, so we do not have to reimplement the game logic on the client side.

Reply:
> We want as much logic as possible on the server side, so we do not have to reimplement the game logic on the client side.

Totally agree